index.html

```diff
@@ -124,30 +118,27 @@ <h3>Full Name</h3>
             <div class="custom-card-wrapper col-12 col-md-6 col-lg-4">
               <div class="custom-card">
                 <div class="avatar-wrapper">
-                  <img src="img/placeholder.jpeg" alt="" class="avatar" />
+                  <img src="img/vasilebirta.jpg" alt="" class="avatar" />
                 </div>
                 <div>
-                  <h3>Full Name</h3>
+                  <h3>Vasile Lucrețiu Birta</h3>
                   <p>
-                    Lorem ipsum dolor sit amet consectetur, adipisicing elit.
-                    Molestiae omnis nobis quo dicta culpa eum exercitationem
-                    provident ex iure pariatur? Tempora sunt accusamus tempore
-                    modi.
+                    Lifetime is a broken challenge in several episodes and IT & JavaScript is a passion that covers all other episodes especially since this passion has taken shape through the prism of vectors like <strong> <a style="text-decoration: none;" href="https://www.itschool.ro/"> ITSchool</a></strong> who have a team of high value instructors who do their job with great passion and one of them is <strong>Răzvan Cîrlugea</strong>, through whom I consider that I have acquired essential knowledge both in JavaScript and in teamwork through Git and GitHub!
```

Comment:
`with great passion and one of them is <strong>Răzvan Cîrlugea</strong>` The phrase becomes too long. You can split it here: `with great passion. One of them is <strong>Răzvan Cîrlugea</strong>`